bridge_ui/src/utils/consts.ts

```diff
@@ -1456,3 +1456,18 @@ export const COLOR_BY_CHAIN_ID: { [key in ChainId]?: string } = {
   [CHAIN_ID_KARURA]: "#FF4B3B",
   [CHAIN_ID_ACALA]: "#E00F51",
 };
+
+export const DISABLED_TOKEN_TRANSFERS: { [key in ChainId]?: string[] } = {
+  [CHAIN_ID_KARURA]: [
+    "0x0000000000000000000100000000000000000081", // aUSD
+  ],
+};
+export const getIsTokenTransferEnabled = (
```

Comment:
should we make this "Disabled" so we don't need the `!`?

Reply:
changed